src/components/Introduction.jsx

```diff
@@ -16,7 +16,11 @@ class Introduction extends Component {
     const buttonURL = `/study-viewer/${studyViewerConfig[0].dataType}`;
     return (
       <div className='introduction'>
-        <div className='h1-typo introduction__title'>{this.props.data.heading}</div>
+        <h1>
+          <div className='h1-typo introduction__title'>
+            {this.props.data.heading}
+          </div>
+        </h1>
```

Comment:
should this be merged to master? NIAID's comment about screen readers is relevant for all Commons

Reply:
same as above